rpm/__init__.py

```diff
@@ -79,21 +93,77 @@ def try_path(path: str) -> bool:
     return False
 
 
+def import_helper(path: Path, suffixes: List) -> None:
+    """
+    Helps import rpm in cases where the shared object file extensions include the python
+    interpreter version
+
+    Args:
+        path (Path): the absolute path to the rpm module
+        suffixes (List): a list of extension suffixes valid for the python interpreter
+
+    Returns:
+        None
+    """
+    try:
+        importlib.reload(sys.modules[__name__])
+    except ModuleNotFoundError as e:
+        if e.name is None:
+            raise
+        logger.debug(
+            f"Failed to import {e.name} from {path}, "
+            "looking for extensions with valid suffixes"
+        )
+        stem = path / e.name.split(".")[1]
+        for suffix in suffixes:
+            so = Path(str(stem) + suffix)
+            logger.debug(f"Looking for {so}")
+            if not so.exists():
+                continue
+            load_module_by_path(e.name, so)
+            importlib.reload(sys.modules[__name__])
```

Comment:
I don't think this is necessary, it causes the module to be executed for a second time.

Reply:
ok nice, confirmed it's not needed